src/adagrams.js

```diff
@@ -1,15 +1,67 @@
-export const drawLetters = () => {
-  // Implement this method for wave 1
-};
+import { letterPool, generateLetters, getRandomLetter } from "helper";
 
-export const usesAvailableLetters = (input, lettersInHand) => {
-  // Implement this method for wave 2
-};
+class Adagrams {
+  constructor(letterPool) {
+    this.letterPool = letterPool;
+  }
```

Comment:
Great use of classes here! Consider the tradeoffs of instead making a deep copy of the letter pool. In which cases would that be preferable? What would the drawbacks be?